kave/docker-troubleshooting
# Change in Dockerfile to fix the health issue.

## Additions

- Fixed the Dockerfile as main.py has /healthz endpoint which will return the health status as "healthy"

## Removals

-

## Changes

- Dockerfile

## Testing Steps

- Run "docker build -t docker_troubleshoot .
- docker run -it -d docker_troubleshoot 
- docker ps
The above command should return status "Healthy"


## Notes

-
